### DISCUSSION TITLE
Configure villager cure time and powdered snow collect chance

### DISCUSSION THREAD

**Discussion by MihitMittal** [PurpurMC/Purpur#547] [__Completed__] [answered]
Hello, I'd like to suggest that admins are able to control how long villagers take to cure after feeding golden apples and change the chance of powdered snow to collect in cauldrons in purpur.yml

**Comment by encode42** [comment, marked as answer]
Powdered snow collection chances closed in https://github.com/pl3xgaming/Purpur/commit/3c451101ac067d15057d86ccc8a3087b054dfcc2, the speed in which a cauldron fills with rain or snow still depends on the random tick speed, though the chances can be configured

Zombie villager cure times closed in https://github.com/pl3xgaming/Purpur/commit/b9480acad46cf6e52c63fbb3bc7cacd2325b0db9
